parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/PLSQL.g4

```diff
@@ -280,9 +284,37 @@ autonomousTransPragma
     ;
 
 dmlEventClause
-    : dmlEventElement (OR dmlEventElement)* ON viewName
+    : (dmlEventElement (OR dmlEventElement)* ON viewName)
+    | (nonDmlEventElement (OR nonDmlEventElement)* ON (DATABASE | (schemaName DOT_)? SCHEMA))
     ;
 
 dmlEventElement
     : (DELETE | INSERT | UPDATE) (OF LP_ columnName (COMMA_ columnName)* RP_)?
     ;
+
+nonDmlEventElement
+    : ALTER
```

Comment:
Can you provide a grammar diagram reference here?

Reply:
here is the reference:
https://docs.oracle.com/cd/B13789_01/server.101/b10759/statements_7004.htm

<img width="1726" alt="Screenshot 2024-01-01 at 13 12 59" src="https://github.com/apache/shardingsphere/assets/14249648/53d88b82-1085-46b9-8acd-130696a0da2f">

Please let me know, if you have any concern.

Reply:
```
CREATE TRIGGER log_errors
  AFTER SERVERERROR ON DATABASE
  BEGIN
    IF (IS_SERVERERROR (1017)) THEN
      NULL;  -- (substitute code that processes logon error)
    ELSE
      NULL;  -- (substitute code that logs error code)
    END IF;
  END;
```

Thank you for providing the document reference. It seems that some parts of SQL should be handled by the `system_trigger` syntax parse.
![2024-01-01_14-29](https://github.com/apache/shardingsphere/assets/106047330/a8653025-47f6-4386-ac28-a8fb67a2bf87)
Ref: https://docs.oracle.com/en/database/oracle/oracle-database/19/lnpls/CREATE-TRIGGER-statement.html#GUID-AF9E33F1-64D1-4382-A6A4-EC33C36F237B__BABHEFGE

Reply:
Thanks for your suggestion, @zihaoAK47, i have updated the changes, could you please take a look?